policyengine_us/variables/gov/ed/pell_grant/efc/pell_grant_dependent_assets.py

```diff
@@ -0,0 +1,8 @@
+from policyengine_us.model_api import *
+
+
+class pell_grant_dependent_assets(Variable):
```

Comment:
Do we need assets separately for dependents? Or can we filter using the dependent variable since it's person level?